bin/transform

```diff
@@ -95,13 +95,18 @@ def parse_geographic_columns(gisaid_data: pd.DataFrame) -> pd.DataFrame:
     Expands the string found in the column named `location` in the given
     *df*, creating four new columns. Returns the modified ``pd.DataFrame``.
     """
-    geographic_data = gisaid_data['location'].str.split('\s*/\s*', expand=True)
-
-    gisaid_data['region']      = geographic_data[0].str.strip()
-    gisaid_data['country']     = geographic_data[1].str.strip()
-    gisaid_data['division']    = geographic_data[2].str.strip()
-    gisaid_data['location']    = geographic_data[3].str.strip()
-
+    geographic_data = gisaid_data['location'].str.split(r'\s*/\s*', expand=True)
+    location_columns = ['region', 'country', 'division', 'location']
+
+    for column, index in zip(location_columns, range(len(location_columns))):
+        gisaid_data[column] = geographic_data[index] \
+            .str.replace('_', ' ') \
+            .str.strip() \
+            .str.title()
```

Comment:
Do we want to write a custom `title()` function that doesn't capitalize articles such as the, of, de, do, etc. or are we okay with manually correcting those?

Reply:
hmm yeah we definitely don't want to capitalize those, nor do we want to manually correct them. I had thought that title case did automatically not capitalize those

Reply:
There are Python libraries like https://github.com/ppannuto/python-titlecase that try to be smart about title casing, but it only works for English words.

Reply:
I see. Ok, then yes, it would be preferable maybe to have a function to specify this, as it will definitely be not great to have to fix those manually

Reply:
Another perhaps simpler option than finding all the non-English articles and states in locations (e.g. `Madison WI`) could be to cast to title case only if the incoming string is in all uppercase.

Reply:
I pushed a new commit that only casts resolution names that are already in uppercase to title case.